wsl-builder/prepare-assets/go.mod

```diff
@@ -4,7 +4,9 @@ go 1.16
 
 require (
 	github.com/spf13/cobra v1.1.3
-	github.com/termie/go-shutil v0.0.0-20140729215957-bcacb06fecae // indirect
+	github.com/termie/go-shutil v0.0.0-20140729215957-bcacb06fecae
+	github.com/ubuntu/WSL-DsitroLauncher/wsl-builder/common v0.0.0-00010101000000-000000000000
 	gopkg.in/gographics/imagick.v2 v2.6.0
-	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
 )
+
+replace github.com/ubuntu/WSL-DsitroLauncher/wsl-builder/common => ../common
```

Comment:
Maybe add a comment to say "We always want to use the latest local version". Well done finding that out btw for someone not familar with Go, and I guess then, Go modules… :)

Reply:
It bit me until I figured out how to fix. When the pain comes, we tend to find solutions. :smile: .
I'll improve the comments.